vendor/github.com/liranr23/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -722,12 +722,30 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 }
 
 func makePopulatePodSpec(pvcPrimeName, secretName string) corev1.PodSpec {
+	nonRoot := true
+	allowPrivilageEscalation := false
+	// QEMU sub GID
+	user := int64(107)
```

Comment:
as it also appears [here](https://github.com/kubev2v/forklift/blob/main/pkg/controller/plan/kubevirt.go#L1149) for the virt-v2v container, how about making it a reusable const?

Reply:
it's within the lib, but we can make a const there.